env/blog.env

```diff
@@ -0,0 +1,4 @@
+LD_PROJECT_TYPE="demo"
```

Comment:
Shouldn't that be `blog`? referring to: https://github.com/drunomics/lupus-decoupled-project/pull/17/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R105

Reply:
yes, thx!